src/styles/blocks/stars--1.css

```diff
@@ -0,0 +1,9 @@
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(../../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:last-child {
+  margin-right: 0;
```

Comment:
The `margin-right: 0;` style seems to be a common style for the last `.stars__star` in the stars block. It would be more efficient to define it once in the general `stars.css` instead of repeating it in each `stars--n.css` file.